src/etc/inc/nat64.inc

```diff
@@ -0,0 +1,164 @@
+<?php
+/*
+ * nat64.inc
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2004-2013 BSD Perimeter
+ * Copyright (c) 2013-2016 Electric Sheep Fencing
+ * Copyright (c) 2014-2020 Rubicon Communications, LLC (Netgate)
+ * Copyright (c) 2020 Nick Whaley
+ * All rights reserved.
+ *
+ * originally part of m0n0wall (http://m0n0.ch/wall)
+ * Copyright (c) 2003-2006 Manuel Kasper <mk@neon1.net>.
+ * All rights reserved.
```

Comment:
Same question here, if you copied content from a file that already had this copyright and changed just a few of it, OK, if you changed like 80/90% then you can remove it